src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started/introduction-new-relic-cordova.mdx

```diff
@@ -50,15 +56,61 @@ Before installing, make sure you have fulfilled the [prerequisites](#prerequisit
    cordova plugin add https://github.com/newrelic/newrelic-cordova-plugin.git --variable IOS_APP_TOKEN="{ios-app-token}" --variable ANDROID_APP_TOKEN="{android-app-token}"
    ```
 5. Deploy your app.
-6. View your performance data from the [mobile monitoring UI](/docs/mobile-monitoring/mobile-monitoring-ui).
+6. View your performance data from the [mobile monitoring UI](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-summary-page#summary-page/).
 
 ## Update the plugin [#update]
 
 Easily update the New Relic Cordova plugin to the latest released version using the following command:
 
+  ```
+  cordova plugin update
+  ```
+
+## Ionic Native Install
+
+  ``` shell
+   ionic cordova plugin add https://github.com/newrelic/newrelic-cordova-plugin.git --variable IOS_APP_TOKEN="{ios-app-token}" --variable ANDROID_APP_TOKEN="{android-app-token}"
+
+   npm install @awesome-cordova-plugins/newrelic
+
+  ```
+
+## API calls [#api-calls]
+
+Our Cordova Plugin utilizes the same API calls as our iOS and Android SDK agents. Please refer to the[New Relic iOS SDK doc](/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api) or [Android SDK doc](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api).
+
+For JavaScript errors, you can call `  NewRelic.recordError(error)` as follows:
+```js
+
+  NewRelic.recordError(error);
+
 ```
-cordova plugin update
-```
+
+## JavaScript errors [#javascript-errors]
+
+The Cordova Plugin creates custom events for JavaScript errors and reports them to New Relic. In the UI, you can track these JavaScript error events with a custom dashboard.
+
+To create a custom dashboard:
+1. Go to [one.newrelic.com](https://one.newrelic.com/all-capabilities).
+2. Click **Query builder**.
+3. Run this query:
+
+  ```sql
+  SELECT * FROM `JS Errors`
+  ```
+4. Click **Add to dashboard**.
+<img
```

Comment:
we're getting an html parsing error, sometimes its due to spacing/indentation weirdities, adding a new line here _might_ help with this

```suggestion

<img
```